frontend/src/providers/AuthProvider.vue

```diff
@@ -106,7 +115,9 @@ const getFamLoginUser = (idToken: CognitoIdToken): FamLoginUser => {
 const handlePostLogin = async () => {
     try {
         isLoading.value = true;
-        const cognitoUser: CognitoUser = await Auth.currentAuthenticatedUser();
+        const cognitoUser: CognitoUser = await Auth.currentAuthenticatedUser({
+            bypassCache: true,
```

Comment:
I like having a comment explaining why this setting is necessary.

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1650/commits/c49b70ef45c87f89834f3a013628f7a96484e366